bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/SonosAudioSink.java

```diff
@@ -115,7 +114,11 @@ public void process(AudioStream audioStream)
                 logger.warn("We do not have any callback url, so Sonos cannot play the audio stream!");
             }
         } else {
-            IOUtils.closeQuietly(audioStream);
+            try {
+                audioStream.close();
```

Comment:
I once again have to jump into the discussion if we really have to close the passed `AudioStream` in `AudioSink` implementations. IIRC Chromecast and Kodi do not close it and I am not aware of a resource leak. Maybe we can find a final decision on it in https://github.com/openhab/openhab-core/issues/532.